pupil-spa/src/index.html

```diff
@@ -27,6 +26,16 @@
       var electron = '<scr' + 'ipt>require(__dirname + "/../../app.asar/lib/renderer");</scr' + 'ipt>';
       $('body').append(electron);
     }
+    // Disable Zoom out in Safari on iOS and macOS (reverse pinch)
+    document.addEventListener('gesturestart', function (e) {
+      e.preventDefault();
+    });
+    // Disable Zoom on Chrome, Opera and FF on the touchpad
+    document.addEventListener('wheel', function (e) {
```

Comment:
is `user-scalable` attribute supposed to also disable pinch zoom on laptop trackpads?
I read somewhere that it should be sufficient but haven't tested it myself.

Reply:
No, it doesn't.  `viewport` is for mobile.